src/Unosquare.RaspberryIO.Peripherals/Accelerometer GY521/AccelerometerGY521.cs

```diff
@@ -0,0 +1,162 @@
+﻿namespace Unosquare.RaspberryIO.Peripherals
+{
+    using System;
+    using System.Threading;
+    using Unosquare.RaspberryIO.Abstractions;
+    using Unosquare.RaspberryIO.Abstractions.Native;
+
+    /// <summary>
+    /// Implements settings for GY-521 accelerometer peripheral.
+    /// </summary>
+    public sealed class AccelerometerGY521 : IDisposable
+    {
+        private const int PwrMgmt1 = 0x6b;
+        private const int GyroConfig = 0x1b;
+        private const int AcclConfig = 0x1c;
+        private readonly ushort[] AFSSel = new ushort[] { 16384, 8192, 4096, 2048 };
+        private readonly double[] FSSel = new double[] { 131.0, 65.5, 32.8, 16.4 };
+        private readonly Thread ReadWorker;
+        private readonly TimeSpan ReadTime = TimeSpan.FromSeconds(0.5);
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AccelerometerGY521"/> class.
+        /// </summary>
+        /// <param name="device"> i2C device. </param>
+        public AccelerometerGY521(II2CDevice device)
+        {
+            Device = device;
+            ReadWorker = new Thread(Run);
+        }
+
+        /// <summary>
+        /// Occurs when [data available].
+        /// </summary>
+        public event EventHandler<AccelerometerGY521EventArgs> DataAvailable;
+
+        /// <summary>
+        /// Retrieves the I2C Device.
+        /// </summary>
+        public II2CDevice Device { get; }
+
+        /// <summary>
+        /// Gets a value indicating whether this instance is running.
+        /// </summary>
+        /// <value><c>true</c> if this instance is running; otherwise, <c>false</c>.</value>
+        public bool IsRunning { get; private set; }
+
+        /// <summary>
+        /// Starts the instance.
+        /// </summary>
+        /// <param name="gyroSens">The gyroscope sensitivity factor.</param>
+        /// <param name="acclSens">The accelerometer sensitivity factor.</param>
+        public void Start(FSSEL gyroSens, AFSSEL acclSens)
+        {
+            // Reset sensor sleep mode to 0.
+            Device.WriteAddressByte(PwrMgmt1, 0);
+
+            Device.WriteAddressByte(GyroConfig, (byte)gyroSens);
+            Device.WriteAddressWord(AcclConfig, (byte)acclSens);
```

Comment:
AccelConfig is an 8bit register, you are writing a word.
you need to set the appropriate bit flags, you are missing the bit shifting.